pyInterface/package/_amplitude.py

```diff
@@ -7,7 +7,7 @@
 
 def calcAmplitude(inputFileName,
                   keyFileName,
```

Comment:
`keyFileName` could be replaced by `waveName` as the key file is no longer used in this function